### PR TITLE
Use locale for mig times and display with ellipses

### PR DESCRIPTION
**Pull request by gildub**
Replaces `formatTimeStamp()` to use Date automated conversion to locale.

times-tz

Resolves https://github.com/konveyor/virt-ui/issues/185

### CODE REVIEW

**Comment by mturley** [comment]
We just need to add a `z` to the end of the format string, which we need the `advancedFormat` dayjs plugin for: https://day.js.org/docs/en/plugin/advanced-format

**Comment by gildub** [comment]
@mturley, not working with `z` but I might have missed something.

**Comment by mturley** [comment]
@gildub I think you also need to extend the `dayjs/plugin/advancedFormat` plugin

File: src/app/common/helpers.ts [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import dayjs from 'dayjs';
+import advancedFormat from 'dayjs-ext/plugin/advancedFormat';
+import timeZone from 'dayjs-ext/plugin/timeZone';
```
Comment (mturley) [inline src/app/common/helpers.ts:3]: @gildub based on the example [here](https://day.js.org/docs/en/plugin/timezone) I think the problem is that you are using a capital Z in `timeZone` and instead you need `timezone`.

Comment (gildub) [reply src/app/common/helpers.ts]: Thanks. Meanwhile still not working, just showing z character.

Comment (mturley) [reply src/app/common/helpers.ts]: That's really weird. I'll take a look when I get a chance. I'm also happy to use something other than dayjs in this repo if we want to switch, we don't use it for much. Someone else in UXD recommended https://date-fns.org/

File: src/app/common/helpers.ts [hunk 2]
```diff
@@ -1,7 +1,14 @@
 import dayjs from 'dayjs';
+import advancedFormat from 'dayjs-ext/plugin/advancedFormat';
```
Comment (mturley) [inline src/app/common/helpers.ts:2]: @gildub sorry I missed this before, you were importing from the wrong package here. I made this change locally and it works.

```suggestion
import advancedFormat from 'dayjs/plugin/advancedFormat';
```

Comment (gildub) [reply src/app/common/helpers.ts]: Thank you, I've mended it.

**Comment by gildub** [comment]
@vconzola, please review.

**Review by mturley — APPROVED** [review]

**Comment by mturley** [comment]
@vconzola I think this matches what you already described, I'll go ahead and merge, if you need changes we can follow up.

**Comment by vconzola** [comment]
Yeah, this looks good.

**Pull request merged**